docker-compose.yml

```diff
@@ -1,6 +1,8 @@
 version: "3.8"
 services:
   proxy:
+    restart: always
```

Comment:
das docker-compose file in deployment kann jetzt weg, oder?

Reply:
ist weg

Reply:
perfekt